lms/djangoapps/support/static/support/jsx/entitlements/data/reducers/error.js

```diff
@@ -0,0 +1,16 @@
+import { errorActions } from '../actions/constants';
+
+const error = (state = '', action) => {
```

Comment:
It feels a little weird to use empty string as the default state instead of `null` or `undefined`?